src/lib/structures/game/RoundData.ts

```diff
@@ -8,6 +8,7 @@ export class RoundData {
 	public readonly primaryArtistGuessers: Snowflake[];
 	public readonly primaryArtist: string;
 	public readonly passedPlayers = new Set<Snowflake>();
+	public readonly startTime: number;
```

Comment:
I'm sure its probably an insignificant amount of time, but I had put it at the end of the constructor so that its as close as possible to when the song actually starts playing.

Reply:
I ran a quick performance test, and both `Date.now()`s would be executed in the same millisecond, so it won't be an issue.